packages/services-store/src/actions/__tests__/profileAPI.test.ts

```diff
@@ -50,6 +56,91 @@ describe('Redux actions for `ProfileAPI`', () => {
     ]);
   });
 
+  it('dispatches the action of error in monitoring user authentication status', async () => {
+    ProfileAPI.getUserStatus.mockRejectedValue(new Error('error-getuserstatus'));
+    const store = mockStore();
+    let caught;
+    try {
+      await store.dispatch(loadUserStatus(MASTHEAD_AUTH_METHOD.DEFAULT));
+    } catch (error) {
+      caught = error;
+    }
+    expect(caught?.message).toBe('error-getuserstatus');
+    expect(convertValue(store.getActions())).toEqual([
+      {
+        type: PROFILE_API_ACTION.SET_REQUEST_USER_STATUS_IN_PROGRESS,
+        request: 'PROMISE',
+      },
+      {
+        type: PROFILE_API_ACTION.SET_ERROR_REQUEST_USER_STATUS,
+        error: 'error-getuserstatus',
+      },
+    ]);
+
+    try {
+      await store.dispatch(loadUserStatus(MASTHEAD_AUTH_METHOD.COOKIE));
+    } catch (error) {
+      caught = error;
+    }
+    expect(caught?.message).toBe('error-getuserstatus');
+    expect(convertValue(store.getActions())).toEqual([
+      {
+        type: PROFILE_API_ACTION.SET_REQUEST_USER_STATUS_IN_PROGRESS,
+        request: 'PROMISE',
+      },
+      {
+        type: PROFILE_API_ACTION.SET_ERROR_REQUEST_USER_STATUS,
+        error: 'error-getuserstatus',
+      },
+    ]);
+
+    try {
+      await store.dispatch(loadUserStatus(MASTHEAD_AUTH_METHOD.DOCS_API));
+    } catch (error) {
+      caught = error;
+    }
+    expect(caught?.message).toBe('error-getuserstatus');
+    expect(convertValue(store.getActions())).toEqual([
+      {
+        type: PROFILE_API_ACTION.SET_REQUEST_USER_STATUS_IN_PROGRESS,
+        request: 'PROMISE',
+      },
+      {
+        type: PROFILE_API_ACTION.SET_ERROR_REQUEST_USER_STATUS,
+        error: 'error-getuserstatus',
+      },
+    ]);
+  });
+});
+
+describe('Redux actions for cloud implementation of `ProfileAPI`', () => {
+  it('dispatches the action to set user authentication status', () => {
+    const store = mockStore();
+    store.dispatch(setUserStatus({ user: 'test.user@ibm.com' }));
+    expect(store.getActions()).toEqual([
+      {
+        type: PROFILE_API_ACTION.SET_USER_STATUS,
+        request: { user: 'test.user@ibm.com' },
+      },
+    ]);
+  });
+
+  it('dispatches the action to get user authentication status', async () => {
+    ProfileAPI.getUserStatus.mockResolvedValue({ user: CLOUD_UNAUTHENTICATED_STATUS });
```

Comment:
since you're testing the cloud authentication methods here, you'll have to use the cloud specific methods - take a look at the ProfileAPI `loadUserStatus()` function:
```switch (authMethod) {
      case MASTHEAD_AUTH_METHOD.COOKIE:
        promiseStatus = ProfileAPI.checkCloudCookie();
        break;
      case MASTHEAD_AUTH_METHOD.DOCS_API:
        promiseStatus = ProfileAPI.checkCloudDocsAPI();
        break;
      default:
        promiseStatus = ProfileAPI.getUserStatus();
```
so just need to reference those methods:
```
  it('dispatches the action to get Cloud cookie user authentication status', async () => {
    ProfileAPI.checkCloudCookie.mockResolvedValue({ user: CLOUD_UNAUTHENTICATED_STATUS });
    const store = mockStore();
    await store.dispatch(loadUserStatus(MASTHEAD_AUTH_METHOD.COOKIE));
    expect(convertValue(store.getActions())).toEqual([
      {
        type: PROFILE_API_ACTION.SET_REQUEST_USER_STATUS_IN_PROGRESS,
        request: 'PROMISE',
      },
      {
        type: PROFILE_API_ACTION.SET_USER_STATUS,
        request: { user: CLOUD_UNAUTHENTICATED_STATUS },
      },
    ]);
  });
  
  it('dispatches the action to get Cloud Docs API user authentication status', async () => {
    ProfileAPI.checkCloudDocsAPI.mockResolvedValue({ user: CLOUD_UNAUTHENTICATED_STATUS });
    const store = mockStore();
    await store.dispatch(loadUserStatus(MASTHEAD_AUTH_METHOD.DOCS_API));
    expect(convertValue(store.getActions())).toEqual([
      {
        type: PROFILE_API_ACTION.SET_REQUEST_USER_STATUS_IN_PROGRESS,
        request: 'PROMISE',
      },
      {
        type: PROFILE_API_ACTION.SET_USER_STATUS,
        request: { user: CLOUD_UNAUTHENTICATED_STATUS },
      },
    ]);
  });

```